src/Interpreter.test.js

```diff
@@ -141,6 +142,58 @@ test('Do a command with a program', (done) => {
 });
 
 test('Doing an unknown command rejects with Error', () => {
-    const interpreter = new Interpreter();
+    const interpreter = new Interpreter(()=>{});
     return expect(interpreter.doCommand('unknown-command')).rejects.toThrow('Unknown command: unknown-command');
 });
+
+test('Use the callback function on continueRun', () => {
```

Comment:
Tests calling `run()` need to be async as `run()` is async: add a parameter to the test function called `done` and call `done()` at the end of the assertions to indicate to the test framework that the async test has completed.